### PR TITLE
Copy update on /support

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Rebuild `/support` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/support
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#)

## Issue / Card

Fixes [#3365](https://github.com/canonical-web-and-design/web-squad/issues/3365)

### CODE REVIEW

**Comment by pmahnke** [comment]
@lechsandecki @bethcollins92 I have asked a number of questions in the copy doc that need to be reviewed before we qa this.  Sorry

**Comment by lechsandecki** [comment]
Thank you, Peter. I just replied to your comments.

**Comment by bethcollins92** [comment]
@pmahnke Ready for review

File: templates/support/index.html [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
+      <h4>Security, support and managed open source for enterprises</h4>
+      <p>Future-proof the full stack, from the data centre to containers to the database, LMA, server and cloud-native applications with Canonical.</p>
+      <p>Ubuntu Advantage is a single, per-node package of the most comprehensive enterprise security and support for open source infrastructure and applications, with managed service offerings available.</p>
       <p>
-        <a class="p-link--inverted" href="/advantage">Get UA Infrastructure Essential free for personal use</a>, or
-      </p>
-      <p>
-        <a href="https://buy.ubuntu.com" class="p-button--positive"><span class="p-link--external">Buy online</span></a>
-        <a href="https://assets.ubuntu.com/v1/1a8fb1b3-UA-I_datasheet_2019-Oct.pdf" class="p-button--neutral">Read the datasheet</a>
+        <a href="/support/contact-us?product=support-overview" class="p-button--positive">Contact us</a>
```
Comment (pmahnke) [inline templates/support/index.html:18]: this should have a class to open the modal

File: templates/support/index.html [hunk 2]
```diff
@@ -36,319 +34,1032 @@ <h4>Open Source for the enterprise</h4>
...
+        <li class="p-list__item is-ticked">Security patching for more than 2,500 packages in Ubuntu Main repository (after 5 years of free LTS security patching)</li>
+        <li class="p-list__item is-ticked">Up to 10 years support for Ubuntu LTS</li>
+        <li class="p-list__item is-ticked">Technical support for OpenStack, KVM, MAAS</li>
+      </ul>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <p>
+        <a href="https://buy.ubuntu.com/">Supported applications&nbsp;&rsaquo;</a>
```
Comment (pmahnke) [inline templates/support/index.html:197]: this should just be an anchor link to the apps support section, no?

File: templates/support/index.html [hunk 3]
```diff
@@ -36,319 +34,1032 @@ <h4>Open Source for the enterprise</h4>
...
-          <p class="p-matrix__desc">Get quickly up and running with managed OpenStack, managed Kubernetes, or a combination of both.</p>
+          <h3 class="p-matrix__title p-heading--four">Managed Open Source</h3>
+          <p class="p-matrix__desc">Get up and running in no time with <a href="/openstack/managed">Managed OpenStack</a>, <a href="kubernetes/managed">Managed Kubernetes</a> and <a href="managed">Managed Apps</a> offerings.</p>
         </div>
       </li>
     </ul>
+  </div>
+</section>
+
+<section class="p-strip">
```
Comment (pmahnke) [inline templates/support/index.html:252]: needs an id for the anchor

File: templates/support/index.html [hunk 4]
```diff
@@ -36,319 +34,1032 @@ <h4>Open Source for the enterprise</h4>
...
+          </td>
+          <td class="u-align--center">
+            <div class="lazy-loaded">
+              <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg">
+            </div>
+          </td>
+        </tr>
+        <tr>
+          <td>Ubuntu Universe</td>
+          <td class="u-align--center">X</td>
```
Comment (pmahnke) [inline templates/support/index.html:696]: these X's should be `&ndash;`s

**Review by pmahnke — CHANGES_REQUESTED** [review]
suggestions in line also, I think we need to redo the apps section... I will make a comment in a second

**Comment by bethcollins92** [comment]
Ready for re-review

**Comment by spencerbygraves** [comment]
@bethcollins92 I think you're missing the mongodb logo:

image

**Comment by pmahnke** [comment]
@spencerbygraves it isn't missing, look at /managed... we were told not to use it.

**Review by pmahnke — APPROVED** [review]
+1

**Comment by spencerbygraves** [comment]
>we were told not to use it.

Ok, fair enough.

**Pull request merged**